src/components/content/order/migrate/SelectMigrateTarget.tsx

```diff
@@ -0,0 +1,173 @@
+    };
+
+    const getRegionList = (
+        e: RadioChangeEvent,
+        selectCsp: UserOrderableServiceVo.csp,
+        selectServiceHostingType: UserOrderableServiceVo.serviceHostingType,
+        selectArea: string,
+        userOrderableServices: UserOrderableServiceVo[] | undefined
+    ) => {
+        let regions: Regions[] = [];
+        const regionList: Regions[] = getRegionDropDownValues(
+            selectCsp,
+            selectServiceHostingType,
+            selectArea,
+            userOrderableServices
+        );
+        if (e.target.value === 'csp') {
+            regions = regionList;
+        } else {
+            regionList.forEach((region) => {
+                if (region.value !== currentSelectedService.deployRequest.region.name) {
+                    regions.push(region);
+                }
+            });
+        }
+        return regions;
+    };
+
+    const getCspList = (e: RadioChangeEvent) => {
+        const currentCspList: UserOrderableServiceVo.csp[] = [];
+        if (e.target.value === 'csp') {
+            userOrderableServiceVoList.forEach((userOrderableServiceVo) => {
+                if (
+                    !currentCspList.includes(userOrderableServiceVo.csp) &&
+                    userOrderableServiceVo.csp !== currentSelectedService.csp
+                ) {
+                    currentCspList.push(userOrderableServiceVo.csp);
+                }
+            });
+        } else {
+            currentCspList.push(currentSelectedService.csp);
+        }
+        return currentCspList;
+    };
+
+    const next = () => {
+        stepItem.status = 'finish';
+        setCurrentMigrationStep(MigrationSteps.SelectADestination);
+    };
+
+    const prev = () => {
+        stepItem.status = 'wait';
+        setCurrentMigrationStep(MigrationSteps.ExportServiceData);
+    };
+
+    return (
+        <>
+            <>
+                <Form form={form} layout='vertical' autoComplete='off' onFinish={next} validateTrigger={['next']}>
+                    <div className={'order-select-migrate-target'}>
+                        <div className={'order-param-item-left'} />
+                        <div className={'order-param-item-content'}>
+                            <Form.Item
+                                name='Change Region/Cloud'
+                                label='Change Regions/Cloud'
+                                rules={[
+                                    {
+                                        required: true,
+                                        message: 'Please select migration between regions or migration between clouds',
+                                    },
+                                ]}
+                            >
+                                <Radio.Group onChange={onChange} value={target}>
+                                    <Radio value={'region'}>Regions</Radio>
```

Comment:
Text for both the buttons should be ->
- Migrate service to a different region offered by the same cloud service provider. 
- Migrate service to a different cloud service provider.

Also put the options one below the other.